.pre-commit-config.yaml

```diff
@@ -29,7 +29,12 @@ repos:
       - id: check-symlinks
       - id: check-case-conflict
       - id: check-docstring-first
-      - id: check-json
+      - id: pretty-format-json
+        args:
+          - "--autofix"
+          - "--indent=4"
```

Comment:
Could we use 2 instead of 4?

Reply:
Works for me! I saw examples of both and picked one. I'll update this shortly.